daily_jobs.py

```diff
@@ -132,30 +133,28 @@ def add_zoom_registrants_to_mailgun():
     meetings = common.all_meetings("me")
 
     # Filter by past meetings
-    past_lrc = [
+    time_now = datetime.datetime.now(tz=pytz.UTC)
+    time_yesterday = time_now - datetime.timedelta(days=60)
+    past_meetings = [
         meeting for meeting in meetings
-        if ("Long Range" in meeting['topic']
-            and pandas.to_datetime(meeting['start_time']) < datetime.datetime.now(tz=pytz.UTC)
-           )
+        if time_yesterday < pandas.to_datetime(meeting['start_time']) < time_now
     ]
 
-    # Restrict to past 20 meetings (not likely to have more than 20 in a day)
-    past_lrc = past_lrc[-20:]
-
-    for lrc in past_lrc:
+    for meeting in past_meetings:
         # Get registrants
-        lrc_registrants = common.meeting_registrants(lrc['id'])
-
-        # Filter those who want to sign up for emails
-        member_data = dict(members=json.dumps([
-            dict(address=i['email'], name="{0} {1}".format(i.get('first_name'), i.get('last_name','')))
-            for i in lrc_registrants if (i.get('May we contact you about future Virtual Science Forum events?','') == "Yes")
-        ]))
-
-        api_query(
-            post, f'lists/{announce_list}/members.json',
-            data=member_data
-        )
+        registrants = common.meeting_registrants(meeting['id'])
+
+        if( len(registrants) != 0 ):
```

Comment:
```suggestion
        if registrants:
```

but also we don't need this condition at all (the subsequent code should run as is). We'd be making an empty request to mailgun, but they don't mind.

Reply:
Does it not read much nicer with the explicit len() != 0? 
I am all in for following the style guide though, so if it is better this way let's go for it. 

Also, in PEP8 examples by Guido I see parentheses for if statements, though they are for multilined statements. Is it only for such multilines that ( ) are used?

Reply:
Actually, forget about the 'reading much nicer', I agree that 'if (there are) registrants' is just as good

Reply:
Here's what pep8 has to say about checking for list truthiness:

![image](https://user-images.githubusercontent.com/2069677/107636670-42a0c680-6c6d-11eb-8c80-497bca1a0d49.png)

Superfluous brackets may be used to emphasize grouping of operators, but it isn't the case now.